keras_cv/models/object_detection/retina_net/retina_net.py

```diff
@@ -98,32 +107,33 @@ class RetinaNet(keras.Model):
         label_encoder: (Optional) a keras.Layer that accepts an image Tensor, a
             bounding box Tensor and a bounding box class Tensor to its `call()` method,
             and returns RetinaNet training targets.  By default, a KerasCV standard
-            LabelEncoder is created and used.  Results of this `call()` method
-            are passed to the `loss` object passed into `compile()` as the `y_true`
-            argument.
-        prediction_decoder: (Optional)  A `keras.layer` that is responsible for
-            transforming RetinaNet predictions into usable bounding box Tensors.  If
-            not provided, a default is provided.  The default `prediction_decoder`
-            layer uses a `MultiClassNonMaxSuppression` operation for box pruning.
+            `RetinaNetLabelEncoder` is created and used.  Results of this object's
+            `call()` method are passed to the `loss` object for `box_loss` and
+            `classification_loss` the `y_true` argument.
+        prediction_decoder: (Optional)  A `keras.layers.Layer` that is responsible
+            for transforming RetinaNet predictions into usable bounding box
+            Tensors.  If not provided, a default is provided.  The default
+            `prediction_decoder` layer is a
```

Comment:
Maybe not in this PR, but I think it's worth considering whether this should be renamed to something other than `decoder`.

Since decoding the deltas to boxes happens in-model, the name is now a bit confusing to me.